addons/utm/models/utm_medium.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import fields, models, api
+
+
+class UtmMedium(models.Model):
+    _name = 'utm.medium'
+    _description = 'UTM Medium'
+    _order = 'name'
+
+    name = fields.Char(string='Medium Name', required=True, translate=True)
+    identifier = fields.Char(
+        string='Identifier', readonly=True, index=True, copy=False)
+    active = fields.Boolean(default=True)
```

Comment:
Side question: we may archive mediums ? Not campaigns or sources ? I suppose this is "already like that", but any idea why ?

Reply:
No :/ No idea, good question